picture.php

```diff
@@ -1029,6 +1029,10 @@ function default_picture_content($content, $element_info)
   include( PHPWG_ROOT_PATH.'include/menubar.inc.php');
 }
 
+if (isset($conf['picture_page_banner']) && !$conf['picture_page_banner'])
```

Comment:
you don't need to check is `isset`. It's always set since you have it in `include/config_default.inc.php`

Reply:
> you don't need to check if `isset`. It's always set since you have it in `include/config_default.inc.php`

Right. Probably was a habit as some time ago I fiddled around with (other) config settings in admin.php and there was some situation when config wasn't loaded yet.


> I think it's a good idea. But I would think it's an even better idea to have it manageable on page [Admin>Config>Options>Display]. It's much more complex to implement be seems better to me. What do you think?

Sure, feel free :-)  I have it just as a quick&dirty hack in my gallery. I also don't know how another solution might interact with different themes (seems the banner isn't available in all), "works for me" ...